wallet/src/de/schildbach/wallet/ExploreSyncWorker.kt

```diff
@@ -117,14 +127,21 @@ class ExploreSyncWorker constructor(val appContext: Context, workerParams: Worke
             }
             val merchantCache = mutableListOf<Merchant>()
             var counter = 0
+            var merchantsLoaded = 0
             exploreRepository.getMerchantData(merchantDataSizeDB).collect {
                 counter++
                 merchantCache.add(it)
                 if (merchantCache.size == PAGE_SIZE) {
                     merchantDao.save(merchantCache)
                     merchantCache.clear()
                     log.info("MERCHANT $counter of ${merchantDataSize - merchantDataSizeDB} sync took $tableSyncWatch")
+
+                    delay(10000)
+                    merchantsLoaded += PAGE_SIZE
+                    log.info("{} - {}", merchantsLoaded, merchantDataSize)
+                    syncStatus.setSyncProgress(10.0 + 0.9 * (merchantsLoaded.toDouble() / merchantDataSize) * 100.0)
```

Comment:
I made up a formula where 10% is for ATM's and 90% is for merchants.  If the device gets interrupted at least once time, then this percentage will be incorrect.  

With future improvements in syncing, perhaps we cannot determine a % at all with copying a single sqlite file.

Currently the UI is using an indeterminate progress bar (rotating circle), so these numbers are just for experimentation at this point.  They seemed to go from 0% to 100% in the logs.